### DISCUSSION TITLE
ServerpodRunMode runMode : What is unique of each mode?

### DISCUSSION THREAD

**Discussion by hugo-a-garcia** [serverpod/serverpod#2592] [Q&A] [answered]
Hi
I am not clear in the specific differences between the different runMode settings when running the server. Can someoone explain please?

I grepped ServerpodRunMode  and then runMode in `serverpod/packages` and it seems that the flag at the moment is setting log levels, password access configuration and SSL configuration???

Just trying to understand what happens for a server running on a Linux box under the different modes?

**Comment by vlidholt** [comment, marked as answer]
The main difference is which configuration file is being used when starting Serverpod. You have three different files (`development.yaml`, `staging.yaml`, and `production.yaml`) in your `config` directory.

**Reply by hugo-a-garcia** [reply]
Thank you. I see the conf directory in the examples with the files.

Would it make sense to have `serverpod create ` also create the conf directory with the default values?
